x-pack/test/functional/services/transform/transform_table.ts

```diff
@@ -274,7 +275,7 @@ export function TransformTableProvider({ getService }: FtrProviderContext) {
 
     public async clickTransformRowActionWithRetry(
       transformId: string,
-      action: 'Delete' | 'Start' | 'Stop' | 'Clone' | 'Edit'
+      action: 'Clone' | 'Delete' | 'Edit' | 'Start' | 'Stop' | 'Discover'
```

Comment:
Would be good to
- also add the possible `action` values to `clickTransformRowAction`, which is actually used by the tests.
- adjust `assertTransformRowActions` to include a check for the `Discover` item.
- adjust `assertTransformRowActionEnabled` to also allow checking the `Discover` button.

Reply:
Fixed in 2616008.